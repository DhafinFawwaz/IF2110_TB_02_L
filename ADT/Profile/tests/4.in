4
08123456789;