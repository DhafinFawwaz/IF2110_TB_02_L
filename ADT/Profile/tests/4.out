Enter new phone number: 
| Nama: Novel
| Bio Akun: 
| No HP: 08123456789
| Weton: 